components/microsoft_teams/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "pipedream/microsoft_teams",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.1",
```

[Context](https://pipedream-users.slack.com/archives/C01E5KCTR16/p1654618930866039?thread_ts=1654103252.389289&cid=C01E5KCTR16)